Sources/Sentry/SentryUIViewControllerPerformanceTracker.m

```diff
@@ -38,6 +45,13 @@ - (instancetype)init
 - (void)viewControllerLoadView:(UIViewController *)controller
               callbackToOrigin:(void (^)(void))callbackToOrigin
 {
+    // Since this will be executed for every ViewController,
+    // we should not create transactions for classes that should no be swizzled.
+    if (![self.inAppLogic isClassInApp:[controller class]]) {
```

Comment:
`m`: Can we please add a test for that in `SentryUIViewControllerPerformanceTrackerTests`?

Reply:
> A few comments, but almost LGTM. Can you also please test with the sample app?

I have a PR only focused on UI tests that Im just waiting to have the loadView spans to finish. 
Maybe I could commit that PR in this one and we review this later. What do you think?